mycpp/gc_alloc.h

```diff
@@ -100,10 +100,18 @@ T* Alloc(Args&&... args) {
 
   DCHECK(gHeap.is_initialized_);
 
-  void* place = gHeap.Allocate(sizeof(ObjHeader) + sizeof(T));
+  constexpr size_t num_bytes = sizeof(ObjHeader) + sizeof(T);
+#if MARK_SWEEP
+  int obj_id;
+  bool in_pool;
+  void* place = gHeap.Allocate(num_bytes, &obj_id, &in_pool);
```

Comment:
I know you said you wanted to unify the Allocate interface but it felt like we should wait until we remove cheney to do that?